dissect/target/plugins/os/unix/log/messages.py

```diff
@@ -1,33 +1,35 @@
-import bz2
-import datetime
 import re
-import zlib
-from io import StringIO
+from datetime import datetime
 
+from dissect.target.helpers.fsutil import TargetPath, decompress_and_readlines
 from dissect.target.helpers.record import TargetRecordDescriptor
 from dissect.target.plugin import Plugin, export
 
 MessagesRecord = TargetRecordDescriptor(
     "linux/messages",
     [
         ("datetime", "ts"),
-        ("uri", "filepath"),
         ("string", "daemon"),
         ("varint", "pid"),
-        ("string", "msg"),
+        ("string", "message"),
         ("string", "raw"),
+        ("uri", "source"),
     ],
 )
 
-RE_TS = re.compile(r"(\w+\s\d+\s\d{2}:\d{2}:\d{2})")
+RE_TS = re.compile(r"(\w+\s{1,2}\d+\s\d{2}:\d{2}:\d{2})")
 RE_DAEMON = re.compile(r"^[^:]+:\d+:\d+[^\[\]:]+\s([^\[:]+)[\[|:]{1}")
 RE_PID = re.compile(r"\w\[(\d+)\]")
 RE_MSG = re.compile(r"[^:]+:\d+:\d+[^:]+:\s(.*)$")
 
 
 class MessagesPlugin(Plugin):
     def check_compatible(self):
-        pass
+        return self.target.fs.path("/var/log/syslog").exists() or self.target.fs.path("/var/log/messages").exists()
+
+    @export(record=MessagesRecord)
+    def syslog(self):
```

Comment:
And add a docstring here. Just something simple such as:
```suggestion
    def syslog(self):
    """Return contents of /var/log/messages and /var/log/syslog.
    
    See ``messages`` for more information.
    """
```